index.html

```diff
@@ -245,6 +245,28 @@
 even prior to this change,
 its use was widespread in employment and other non-government interactions.
 
+### Multiple National Credentials
+
+Individuals with more than one nationality, when traveling across international borders,
+have the choice of which credential (passport) to assert.  
+For example, a person is generally required to present a passport
+for the country they entering if they are a citizen of that country,
+even if they hold other passports.
+When entering a country in which they do not hold citizenship,
+individuals with multiple nationalities have the choice of asserting
+whichever nationality is more convenient 
+(for example, one where they would not have to obtain a visa prior to traveling).
+The systems that support digital passport credentials –
+and therefore enable these assertions to move to the digital domain –
+should not behave any differently.
+Some countries already provide web apps (such as Visit Japan Web) that enable people
+to pre-register their passports by entering passport data manually,
+so it's not inconceivable that national authorities will want to evolve these web apps
+to take advantage of digital credentials.
```

Comment:
```suggestion
and national authorities will likely build these web apps
to take advantage of digital credentials when they become available.
```

Reply:
How about that? I think this does require some additional emphasis, as some have pushed back on this risk, especially if they are not focused on cross-border use cases, or they don't think the web currently plays a role here.